front/app/containers/MainHeader/Components/NavbarContent/MobileNavbarContent/FullMobileNavMenu.tsx

```diff
@@ -92,7 +78,8 @@ const MenuItems = styled.ul`
 `;
 
 const StyledTenantLogo = styled(TenantLogo)`
-  margin-bottom: 40px;
+  margin-bottom: 0px;
+  padding: 0px;
```

Comment:
I strongly suspect these styles are not being applied because TenantLogo has no className prop (which is needed for styled component styles to work.

Did you see changes in the UI when you added these? 🤔

Reply:
(So I think this styled component can be removed altogether)